include/subnet_cadmium.hpp

```diff
@@ -97,7 +145,13 @@ class Subnet {
             return bags;
         }
 
-        // time_advance function
+        /**
+         * Function sets next internal transmission time.
+         * If the current state is transmitting then the next
+         * internal time is set to value generated by distribution function
```

Comment:
You may consider saying that the mean is 3.0 and standard deviation as 1.0

Reply:
solved